core/src/main/java/org/jsmart/zerocode/core/kafka/consume/ConsumerLocalConfigs.java

```diff
@@ -36,7 +37,8 @@ public ConsumerLocalConfigs(
             @JsonProperty("pollingTime") Long pollingTime,
             @JsonProperty("cacheByTopic") Boolean cacheByTopic,
             @JsonProperty("filterByJsonPath") String filterByJsonPath,
-            @JsonProperty("seek") String seek) {
+            @JsonProperty("seek") String seek,
+            @JsonProperty("seekToTimestamp") Long seekToTimestamp) {
```

Comment:
Minor suggestion here @a1shadows :
Better to make it "String" (instead of Long), so that we can support the "YYYY-MM-DD...." format too later.

Reply:
Done